debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlLexer.g4

```diff
@@ -0,0 +1,1170 @@
+/*
```

Comment:
Can you add a link to the original source of the file?

Reply:
I added a link to files in git repo